authenticator.js

```diff
@@ -11,6 +11,7 @@ function AuthenticateGithub(opts) {
     debug: true,
     githubHost: config.githubHost,
     githubPathPrefix: '/api/v3',
+    githubOrg: config.githubOrg,
```

Comment:
I know it's been a while, but would it be possible to extend it in order to support multiple organizations? Really don't want to come up with a fork of this repo just because of this restriction.